### DISCUSSION TITLE
Where do we take TRAPI from here?

### DISCUSSION THREAD

**Discussion by edeutsch** [NCATSTranslator/ReasonerAPI#490] [General]
This discussion thread will serve as the voting platform for prioritizing where do we take TRAPI from here for the next 3 months

All regular team representatives/attendees of the TRAPI call are strongly requested to vote. All other Translator participants are welcome to vote.

Voting Scheme:
+2 Highest (heart) - Love love love this, this is what we should do
+1 High (thumb up) - This is something to put some effort into in the next 3 months
0 Medium (none) - Meh, lukewarm
-1 Low Priority (thumb down) - This is not where our priorities should be

Results:
2. 8 pts. Tech debt but no schema changes
3. 4 pts. Manuscript
1. 3 pts. Nothing
4. 3 pts. Tech debt with schema changes
5. -3 pts Major changes

**Comment by edeutsch** [comment]
TRAPI is fine as is. Leave it alone for the next 3.5 months as we work on paying down technical debt of our individual TRAPI 1.5 implementations

Vote for this path forward with an emoji: heart, thumbs up, nothing, thumbs down

**Comment by edeutsch** [comment]
Clean up technical debt but WITHOUT changing the schema (documentation work, cleaning up issues and making a list of future work)

Vote for this path forward with an emoji: heart, thumbs up, nothing, thumbs down

**Comment by edeutsch** [comment]
Instead of changing TRAPI, let's focus our energies writing a journal manuscript to announce it more widely

Vote for this path forward with an emoji: heart, thumbs up, nothing, thumbs down

**Comment by edeutsch** [comment]
Get to work on TRAPI 1.6 but mostly just cleaning up technical debt (minor schema changes such as tweaking required properties, open issues and PRs, etc.)

Vote for this path forward with an emoji: heart, thumbs up, nothing, thumbs down

**Reply by CaseyTa** [reply]
Voting up with the caveat without releasing during this phase

**Comment by edeutsch** [comment]
Get to work on new features for TRAPI 1.6 needed now

Vote for this path forward with an emoji: heart, thumbs up, nothing, thumbs down
